### ISSUE TITLE
Offset of second level menu items in collapsed mode

### ISSUE THREAD

**Issue by PowerGamer1** [kartik-v/yii2-nav-x#14]
When navbar becomes collapsed (when browser window width is less then 767px), second level menu items (SubSubItem1 and SubSubItem2 in the example below) need to have bigger offset from the left in comparison with the first level menu items (SubItem1, SubItem2, SubItem3) which have some default offset set by Bootstrap.

```PHP
	<?php NavBar::begin([
		'options' => [
			'class' => 'navbar navbar-default navbar-static-top',
		],
	]); ?>
		<?= \kartik\nav\NavX::widget([
			'options' => ['class' => 'nav navbar-nav'],
			'items' => [
				[
					'label' => 'RootItem1',
				],
				[
					'label' => 'RootItem2',
					'items' => [
						[
							'label' => 'SubItem1',
						],
						[
							'label' => 'SubItem2',
							'items' => [
								[
									'label' => 'SubSubItem1',
								],
								[
									'label' => 'SubSubItem2',
								],
							],
						],
						[
							'label' => 'SubItem3',
						],
					],
				],
				[
					'label' => 'RootItem3',
				],
			],
		]) ?>
	<?php NavBar::end(); ?>
```

![navbar](https://cloud.githubusercontent.com/assets/8275897/16962962/b91ebb08-4dfc-11e6-8c81-d1a81b3af7d4.png)

The following style needs to be included in the widget to fix the problem:
```CSS
@media (max-width: 767px) {
    .navbar-nav .open .open .dropdown-menu > li > a,
    .navbar-nav .open .open .dropdown-menu .dropdown-header {
        padding-left: 35px;
    }
}
```

**Comment by kartik-v** [comment]
Thanks - can you submit a PR for both the CSS and its minified version.

**Comment by PowerGamer1** [comment]
Updated my pull request with additional fix. Now open SubItem2 will have the proper background color - the same one as open RootItem2 (see screenshot in my first post) for standard Bootstrap navbar classes (`navbar-default` and `navbar-inverse`).

**Comment by PowerGamer1** [comment]
Please also merge https://github.com/kartik-v/yii2-nav-x/pull/16 In my original pull request I forgot to put css folder inside the assets folder.

**Comment by kartik-v** [comment]
Closed via #16.
